### PR TITLE
feat(fe): implemented profiles page

### PR DESCRIPTION
**Pull request by ShadySloth**
implemented profiles page without functionality

### CODE REVIEW

File: src/FE/public/i18n/da.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
   "CONTRIBUTED_HOURLY_RATE": "Kont. timerate",
   "CONTRIBUTED_DAY_RATE": "Kont. dagsrate",
   "CONTRIBUTED_ANNUAL_COST": "Kont. årlige omkostninger",
```
Comment (DanKjaer) [inline src/FE/public/i18n/da.json:62]: Eventuelt lav en annual_cost os, ligesom med hours - Så det matcher?

Comment (DanKjaer) [inline src/FE/public/i18n/da.json:62]: Eventuelt lav en annual_cost os, ligesom med hours - Så det matcher?

**Review by hoeeg79 — CHANGES_REQUESTED** [review]
There are scoll bars on the sidebar at all times. 

Sideways scroll doesn't seem to be working in the profile view, and it doesn't shrink with the window, it seems

**Review by hoeeg79 — APPROVED** [review]
wery nice

**Pull request merged**